docs/quick-starts/framework/nuxt/_get-user-information.mdx

```diff
@@ -22,7 +22,7 @@ When user is signed in, the return value of `useLogtoUser()` will be an object c
   </ul>
   <!-- highlight-end -->
   <!-- Simplified button for sign-in and sign-out -->
-  <a :href="`/sign-${ user ? 'out' : 'in' }`"> Sign {{ user ? 'out' : 'in' }} </a>
+  <nuxt-link :to="`/sign-${ user ? 'out' : 'in' }`"> Sign {{ user ? 'out' : 'in' }} </nuxt-link>
```

Comment:
Don't we need to import `NuxtLink` component first and use `<NuxtLink>` instead of `<nuxt-link>`? Not quite sure about this. Maybe @wangsijie can help me verify it as well

Reply:
Sorry for my late answer, in Nuxt these components are auto imported (see https://nuxt.com/docs/guide/concepts/auto-imports)

```<NuxtLink>``` and ```<nuxt-link>``` are the same in Vue. You could argue that ```<NuxtLink>``` is more easily distinguishable from native HTML elements which I get.

Reply:
Thanks for the clarification. Then I'm good with both. Will merge this PR and move forward.

Thanks again for your contribution.